### PR TITLE
Run e2e tests in CI with latest prebuilt backend binary

### PR DESCRIPTION
**Pull request by thomasballinger**
Getting the old path of an open source backend directory wasn't temporarily broken so I hooked this up to prebuilt binaries.

### CODE REVIEW

**Comment by thomasballinger** [comment]
I ran into issues with using with a convex monorepo or a convex-backend repo for testing:
- with a convex/ repo, `run-local-backend` already has a `--port` flag so runs on 8000, can't be overridden with another `--port` flag
- with a convex-backend repo, `run-local-backend` seems broken because the `simulation` project isn't present

File: test-nextjs/e2e-tests/README.md [hunk 1]
```diff
@@ -15,7 +19,7 @@ you can skip to running the test command at the end.
 CONVEX_LOCAL_BACKEND_PATH=/path/to/your/convex-backend npm run test
 ```
 
-## The manual way
+## The most manual way
```
Comment (dowski) [inline test-nextjs/e2e-tests/README.md:22]: Haha, love it!

File: test-nextjs/playwright.config.ts [hunk 2]
```diff
@@ -38,17 +38,6 @@ export default defineConfig({
       name: "chromium",
       use: { ...devices["Desktop Chrome"] },
     },
-
-    {
-      name: "firefox",
```
Comment (dowski) [inline test-nextjs/playwright.config.ts:43]: Why remove these other browsers?

**Review by dowski — APPROVED** [review]
Cool, nice change.

File: test-nextjs/playwright.config.ts (hunk 2 above)
Comment (thomasballinger) [reply test-nextjs/playwright.config.ts:43]: No good reason, I'll add them back.

**Pull request merged**